packages/base/src/index.ts

```diff
@@ -6,11 +6,16 @@ import { CellDep, HexString, HexNumber, Hash, Script } from "@ckb-lumos/base";
 export interface DeploymentConfig {
   deposition_lock: Script;
   custodian_lock: Script;
+  withdrawal_lock: Script;
   state_validator_lock: Script;
   state_validator_type: Script;
+  sudt_type: Script;
 
   deposition_lock_dep: CellDep;
   custodian_lock_dep: CellDep;
   state_validator_lock_dep: CellDep;
   state_validator_type_dep: CellDep;
+  sudt_type_dep: CellDep;
+
+  l2_finalized_period: BigInt;
```

Comment:
This is not a cell configuration, it should be in Godwoken's config, not here.

Reply:
https://github.com/nervosnetwork/godwoken/blob/a0d93bc8af5ecadfd9491cdf859803b72e120c0c/crates/common/src/lib.rs#L18 We can set it as a hardcoded value like the Rust side for now, then later refactor it to a config variable.

Reply:
updated.